src/node.rs

```diff
@@ -453,6 +456,30 @@ impl MinerStats {
     }
 }
 
+#[derive(StructOpt, Debug)]
+pub struct MinerRevert {
+    #[structopt()]
```

Comment:
does this attribute do anything?

Reply:
I will extend the struct to support specfying port to get ledge info.